homework/index.js

```diff
@@ -109,11 +109,16 @@
           setRepo(repoId);
         });
       })
-      .catch(() => {});
-    // const root = document.getElementById('root');
+      .catch(() => {
+        container.parentNode.removeChild(container);
+        const divError = document.createElement('div');
+        rootContainer.appendChild(divError);
+        divError.className = 'alert-error ';
+        divError.innerHTML = `Sorry, there's something wrong.`;
+      });
   };
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  const HYF_REPOS_URL = 'https://api.github.com/orgs1x/HackYourFuture/repos?per_page=100';
```

Comment:
Was this, to test the error message? You should not have commited  that.  (minor error)

Reply:
Yes, I have fixed it on the next commit, I forgot to put it back as original